components/ListView/ComponentDetailsView.js

```diff
@@ -75,16 +74,17 @@ class ComponentDetailsView extends React.Component {
         this.setState({ componentData: data[0][0] });
         const filteredDependencies = data[0][0].dependencies.filter(dependency => dependency.name !== data[0][0].name);
         this.setState({ dependencies: filteredDependencies });
-        if (this.props.status === "editSelected") {
+        const { status } = this.props;
+        if (status === "editSelected") {
```

Comment:
This looks wrong. `status` is a method parameter, and unrelated to `this.props.status`. Also, why all this excessive aliasing like `const { status } = this.props;` ? This is just noise, makes the code harder to follow, and is prone to namespace conflicts like this one.